admin/tabs/roles/details/RoleDetailsModel.ts

```diff
@@ -0,0 +1,65 @@
+import {roleMembers} from '@xh/hoist/admin/tabs/roles/details/members/RoleMembers';
+import {RolesModel} from '@xh/hoist/admin/tabs/roles/RolesModel';
+import {FormModel} from '@xh/hoist/cmp/form';
+import {TabContainerModel} from '@xh/hoist/cmp/tab';
+import {HoistModel, lookup, managed} from '@xh/hoist/core';
+import {HoistRole} from '../HoistRole';
+
+export class RoleDetailsModel extends HoistModel {
+    @lookup(() => RolesModel) readonly rolesModel: RolesModel;
+
+    @managed readonly formModel: FormModel = this.createFormModel();
+    @managed readonly tabContainerModel = this.createTabContainerModel();
+
+    get selectedRole(): HoistRole {
+        return this.rolesModel.selectedRole;
+    }
+
+    override onLinked() {
+        this.addReaction({
+            track: () => this.selectedRole,
+            run: role => this.formModel.init(role ?? {})
+        });
```

Comment:
Little gotcha here - if selectedRole already set before linked, you won't get your form populated.  I noticed we got into this state in the last demo.  Collapse detail panel (currently persisted, prob shouldn't be) then select a role in the grid then expand detail.